tests/README.md

```diff
@@ -2,46 +2,43 @@
 
 ## Initial Setup
 
-1) Install [PHPUnit](http://phpunit.de/) by following their [installation guide](https://phpunit.de/getting-started.html). If you've installed it correctly, this should display the version:
+From the WooCommerce root directory (if you are using VVV you might need to `vagrant ssh` first), run the following:
 
+1. Install [PHPUnit](http://phpunit.de/) via Composer by running:
     ```
-    $ phpunit --version
+    $ composer install
     ```
 
-2) Install WordPress and the WP Unit Test lib using the `install.sh` script. Change to the plugin root directory and type:
-
+2. Install WordPress and the WP Unit Test lib using the `install.sh` script:
     ```
     $ tests/bin/install.sh <db-name> <db-user> <db-password> [db-host]
     ```
 
-The `<db-password>` will be set as given. Previously, you would have needed to escape certain characters (forward & backward slashes, and ampersand), but install.sh now escapes them when it needs to internally. You may still need to quote strings with backslashes to prevent them from being processed by the shell or other programs.
+You may need to quote strings with backslashes to prevent them from being processed by the shell or other programs.
 
-Sample usages:
+Example:
 
     $ tests/bin/install.sh woocommerce_tests root root
 
-    #  The actual password only has a single backslash, but it's escaped
-	#  to prevent the shell and PHP from treating it as a backspace character
-    $ tests/bin/install.sh woocommerce_tests root 'a\\b/&'
-    #  Previously, the password would have had to be passed as 'a\\\\b\/\&'
+    #  woocommerce_tests is the database name and root is both the MySQL user and its password.
 
 **Important**: The `<db-name>` database will be created if it doesn't exist and all data will be removed during testing.
 
 ## Running Tests
 
 Simply change to the plugin root directory and type:
```

Comment:
Maybe get rid of the word `Simply` as well? It can have a negative implication for the reader if they don't know how to do it.

Reply:
Good point, I just pushed a commit removing the word "simply". Thanks.